vllm/logging_config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "logger": {
+    "structured_log_file_path": "/logs/vllm/application_structured.log",
+    "unstructured_log_file_path": "/logs/vllm/application.log",
```

Comment:
for default, let's consider logging to /tmp

Reply:
I think by default there should not be any files written. vllm doesn't write files right now, so it I think this should stay the same. Whoever wants to have log files could pass some config.  Writing files could slow down the application. If you decide to write files by default the right place is /var/log/vllm.log

Reply:
Shall I move it to /var or /tmp?
We can consider set the default behavior to no-file, no-json. I have added the option in the latest logging_config.json.